src/components/book/Book.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Book = (props) => {
+  const { title } = props;
+  return (
+    <div>
+      I am a book, my title is
+      {title}
+    </div>
```

Comment:
- [ ] Kindly include the author prop also. It should receive not only the title prop but the author prop also.
- [ ] In addition please add a button to remove the book. It would just be a placeholder for now.

Reply:
Done!